job-service-container/pom.xml

```diff
@@ -179,7 +179,7 @@
                 </property>
             </activation>
             <properties>
-                <task.table.deletion.count>30000</task.table.deletion.count>
+                <task.table.deletion.count>10000</task.table.deletion.count>
```

Comment:
This number now corresponds to number of parent tables. For every parent table, we are creating 20 twenty tables(child + parent). So approximately 10k * 20 tables are getting created in total.